tests/step_defs/ui/test_ui_admin.py

```diff
@@ -0,0 +1,51 @@
+# coding=utf-8
+"""Admin UI feature tests."""
+
+__copyright__ = "Copyright (c) 2024, Utrecht University"
+__license__ = "GPLv3, see LICENSE"
+
+import time
+
+from pytest_bdd import (
+    parsers,
+    scenarios,
+    then,
+    when,
+)
+
+from conftest import portal_url
+
+scenarios("../../features/ui/ui_admin.feature")
+
+
+@when(parsers.parse("user opens link to admin page"))
+def ui_admin_access(browser):
+    url = "{}/admin".format(portal_url)
+    time.sleep(10)  # FIXME: In case of slow VM machine
```

Comment:
Is this fixed when running against the Docker setup?

Reply:
removed